protocol/Makefile

```diff
@@ -49,7 +47,8 @@ swift-build-stamp: build-stamp
 	date > $@
 
 clean:
-	rm -rf json/*.json ../go/protocol/keybase_v1.go js/keybase_v1.js swift/*
+	rm -rf json/*.json js/keybase_v1.js swift/* *build-stamp
```

Comment:
should actually be js/* so we get flow-types.js also (nothing to do with this, but it should be)